bin/oref0-pump-loop.sh

```diff
@@ -118,7 +118,7 @@ main() {
 function update_display {
     # TODO: install this globally
     if [ -e /root/src/openaps-menu/scripts/status.js ]; then
-        node /root/src/openaps-menu/scripts/status.js
+        node /root/src/openaps-menu/scripts/status.js ~/myopenaps/settings/profile.json
```

Comment:
All other references to stuff in ~/myopenaps/ use relative paths, as oref0-pump-loop is run from inside ~/myopenaps/ - so I think we can change this to just be `settings/profile.json`.  Want to give that a shot?

Reply:
you're right, i think i did this wrong in the menu code, but i also noticed that the menu goes to get the files directly instead of taking them as parameters, so i'll do the change that way instead. Will close this pr.